main/src/cgeo/geocaching/log/LogCacheActivity.java

```diff
@@ -455,7 +455,7 @@ private void initializeFavoriteCheck() {
         if (ConnectorFactory.getConnector(cache).supportsAddToFavorite(cache, typeSelected)) {
             if (premFavPoints > 0) {
                 favCheck.setVisibility(View.VISIBLE);
-                favCheck.setText(getString(R.string.fav_points_remaining, premFavPoints));
+                favCheck.setText(getString(R.string.fav_points_remaining, String.valueOf(premFavPoints)));
```

Comment:
Why not either:

- fix the string to use `%d` instead of `%s`
- do nothing and let the implicit `.toString()` do the job

rather than calling `String.valueOf` explicitly?